uast/utils.go

```diff
@@ -17,3 +17,34 @@ func Tokens(n *Node) []string {
 	}
 	return tokens
 }
+
+// This returns the cyclomatic complexity for the node. This uses the method of counting one + one
```

Comment:
Godoc for functions start with the name of the function: `CyclomaticComplexity returns the cyclomatic complexity for the node...`.